### PR TITLE
change voice changer masks to act as internals

### PR DESCRIPTION
**Pull request by ghost**
## About the Pull Request

Voice changer masks don't function as internals; this changes that.

Thoughts on balance?

## Why It's Good For The Game

~~Brings the blurb desc. in-line with reality:~~

~~`A face-covering mask that can be connected to an air supply.`~~

It's less useful as a traitor mask, esp. if you're engineer, if it doesn't function as an internals mask.

## Changelog

:cl:
add: voice changer masks now copy the item_flags of the masks they're copying, i.e. internals masks can be used as internals
/:cl:

### CODE REVIEW

**Comment by Tsurupeta** [comment]
It's chameleon mask, you can change its appearance to any... And it will copy item's desc.

File: code/datums/extensions/chameleon.dm [hunk 1]
```diff
@@ -15,7 +15,7 @@
 		chameleon_choices = LAZYACCESS(chameleon_choices_by_type, chameleon_type)
 		if(!chameleon_choices)
 			chameleon_choices = generate_chameleon_choices(chameleon_type)
-			LAZYSET(chameleon_choices_by_type, chameleon_type, chameleon_choices)
```
Comment (CristoMagisto69) [inline code/datums/extensions/chameleon.dm:18]: Linter

**Comment by ghost** [comment]
> It's chameleon mask, you can change its appearance to any... And it will copy item's desc.

Looks like it wasn't copying over item_flags, so they weren't getting the "airtight" etc. copied over, i.e. can't use em as internals

Reverted commit and made the change elsewhere to better reflect the intent.

EDIT: "Why It's Good For The Game" was cheeky nonsense. Real reason is that it's less useful as a traitor mask, esp. if you're engineer.

**Review by Tsurupeta — APPROVED** [review]
sure, what could go wrong

**Comment by Tsurupeta** [comment]
Ok, I am already regretting this, but whatever.

**Comment by ghost** [comment]
@Tsurupeta Yeaaah. My main concern was with balance. 

But it doesn't look like there are any items flags involved with masks besides `ITEM_FLAG_BLOCK_GAS_SMOKE_EFFECT` (smoke),  `ITEM_FLAG_AIRTIGHT` (internals), and `ITEM_FLAG_FLEXIBLEMATERIAL` (consumables?). 

Atleast it's not copying over armor value lists ☠️ or the gas filtration systems.

**Comment by Tsurupeta** [comment]
You changed all of chameleon items to copy flags

**Comment by ghost** [comment]
This was not intended. 

Shoes: `ITEM_FLAG_NOSLIP`, `ITEM_FLAG_SILENT`
Glasses: `null`
Earwear: `null`
Gloves: `ITEM_FLAG_PREMODIFIED` (atleast there's no copying over of `siemens_coefficient`)
Headgear: `ITEM_FLAG_THICKMATERIAL`, `ITEM_FLAG_AIRTIGHT`
Suits: `ITEM_FLAG_THICKMATERIAL` `ITEM_FLAG_PHORONGUARD`

This could've been much worse.

**Comment by ghost** [comment]
I need to slow down, and read Dantom's `The Dream Maker: Designer’s Guide to Worlds BYOND`, before I touch anything else.

**Pull request merged**